Website/DesktopModules/Hotcakes/Providers/DataProviders/SqlDataProvider/02.00.00.SqlDataProvider

```diff
@@ -137,6 +137,61 @@ BEGIN
 END
 GO
 
+IF NOT EXISTS (SELECT 1 FROM {databaseOwner}[{objectQualifier}hcc_HtmlTemplates] WHERE [DisplayName] = N'Daily Online Order Payment Failure Report')
+BEGIN
+    INSERT INTO {databaseOwner}[{objectQualifier}hcc_HtmlTemplates] ([StoreId],[LastUpdatedUtc],[DisplayName],[FromEmail],[TemplateType])
+    VALUES (0, GETDATE(), N'Daily Online Order Payment Failure Report', N'[[Store.ContactEmail]]', 511)
+
+    INSERT INTO {databaseOwner}[{objectQualifier}hcc_HtmlTemplateTranslations] ([HtmlTemplateId],[Culture],[Subject],[Body],[RepeatingSection])
+    VALUES (@@IDENTITY, 'en-US', N'Daily Online Order Payment Failure Report', N'<html>
+    <head>
```

Comment:
@alejoroman0605 Please remove the `<hr>` from the body.  Also, the orders should be an unordered list (`<ul>`).